app/models/preapplication_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PreapplicationService < ApplicationRecord
+  belongs_to :planning_application
+
+  TYPES = %i[
+    written_advice
+    1-2-1_meeting
+    site_visit
+  ].freeze
```

Comment:
Snake or kebab case - no mixed 🙈

Reply:
Also this seems a hostage to fortune to define these in code and not via a configurable list per local authority but MVP, yada.

Reply:
It's not kebab case, it's snake case for "1-2-1 meeting". Though I also hate it. I'll find a better name.

Reply:
> I'll find a better name.

Meeting?